ui/app/templates/components/text-file.hbs

```diff
@@ -4,6 +4,11 @@
       <label class="is-label" data-test-text-label=true>
         {{#if label}}
           {{label}}
+          {{#info-tooltip}}
```

Comment:
I think we'll want an `{{#if helpText}}` around this otherwise it'd render a tooltip with no content.